docs/releasenotes/archive/service/year2017.md

```diff
@@ -6,3 +6,460 @@ description: This is an archive of 2017 Sumo Logic service release notes.
+
+---
+## November 27, 2017 (Search) 
+
+Update - fields operator for defining the order of the columns. Along with choosing which fields are displayed in the results, you can now use the fields operator to order the columns in the result. The order of the columns in the result would be the order you specified with the fields operator.
+
+---
+## November 22, 2017 (Apps) 
+
+New Beta App - The Sumo Logic App for AWS Elastic Load Balancing ULM - Application is now available. It is a unified logs and metrics (ULM) App that gives you visibility into the health of your Application Load Balancer and target groups. Use the preconfigured dashboards to understand the latency, request and host status, threat intel, and HTTP backend codes by availability zone and target group.
+New panels added to the PCI Compliance for CloudTrail App - The PCI Compliance for AWS CloudTrail App is now updated to include the details of create and delete group, added and removed users, and password events in the PCI Req 08 - Account, System Monitoring dashboard; policy operations in the PCI Req 08, 10 - Privileged Activity dashboard; and console root logins in the PCI Req 10 - Login Activity dashboard.
+
+---
+## November 21, 2017 (Apps) 
+
+New Beta App - Sumo Logic has a beta app for Amazon SQS, a unified logs and metrics (ULM) app that provides operational insights into your Amazon Simple Queue Service (SQS) use. The pre-configured dashboards help you monitor the key metrics, view the SQS events for queue activities, and help you plan the capacity of your SQS service.
+
+New Beta App - Sumo Logic has a beta unified logs and metrics (ULM) app for AWS Elastic Load Balancing ULM - Classic. You can use the searches and dashboards to track ELB information on the latency, HTTP backend codes, requests, and host status, to investigate issues in the load balancer.
+
+---
+## November 21, 2017 (Dashboards) 
+
+Bug Fix  - Dashboards with the same name now delete properly.  
+Bug Fix - Drill down metrics name consistent with panel name on dashboard panel. 
+Bug Fix - Dashboards now display properly, even when tabs are closed. 
+Bug Fix - Share dashboards with filters option only available when filters are present. 
+Bug Fix - Clicking the Zoom into Panel option consistently opens a new panel. 
+
+---
+## November 21, 2017 (Metrics) 
+
+Bug Fix -  Update dashboard option for metrics charts is fixed. 
+Update - Metrics Introduction now references new UI layout, error messages updated, and revised and improved metrics query performance.
+Bug Fix - Aggregate group-by key is no longer case sensitive.
+Bug Fix -  You now go to the metrics page after clicking "Exit Setup Wizard" if you are working with a metrics source.
+Bug Fix - Division by zero not shown in metrics time series to improve visualizations.
+Bug Fix - Dashboards shared with“World” now open in incognito mode.    
+Bug Fix - Removing filter refreshes screen. 
```

Comment:
Needs to be a bulleted list

Reply:
It's done as a bulleted list in DocHub, but it's not needed as they are the release information that day. They are not sub-points.

Reply:
Ok! So here you'd just need to put line breaks between each. Otherwise they'll be lumped together as a giant paragraph.